src/components/app.js

```diff
@@ -29,6 +29,7 @@ import {
 } from '../utils/authentication/AuthService';
 
 import Authentication from "./authentication";
+import PageNotFound from "./PageNotFound";
```

Comment:
You have choosen here the nomenclature **OneTwoThree.js** rather than **one-two-three.js**.
It seems we use the second one in our code instead.

Reply:
In the end, I would like to have an organization in this form: components/PageNotFound/index.js in the commons-ui [seems to be recommended](https://medium.com/@alexmngn/how-to-better-organize-your-react-applications-2fd3ea1920f1). But for the moment, for the sake of uniformity in this repo :)...

Reply:
> In the end, I would like to have an organization in this form: components/PageNotFound/index.js in the commons-ui [seems to be recommended](https://medium.com/@alexmngn/how-to-better-organize-your-react-applications-2fd3ea1920f1). But for the moment, for the sake of uniformity in this repo :)...

I hate this layout. This is very unusual and a terrible idea. Everything is called index.js. What a mess in your ide. Nobody do that.